dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -64,14 +68,14 @@
     @Override
     public void createIndices(final DBCollection coll){
         BasicDBObject doc = new BasicDBObject();
-        doc.put(SUBJECT, 1);
-        doc.put(PREDICATE, 1);
+        doc.put(SUBJECT_HASH, 1);
+        doc.put(PREDICATE_HASH, 1);
         coll.createIndex(doc);
```

Comment:
@pujav65 thanks.  

@isper3at clearly a bug.  please add OBJECT_HASH, OBJECT_TYPE_HASH to the first index.